updatehub/src/main.rs

```diff
@@ -114,7 +114,25 @@ async fn client_main(cmd: ClientCommands) -> updatehub::Result<()> {
 
     match cmd {
         ClientCommands::Info(_) => println!("{:#?}", client.info().await),
-        ClientCommands::Log(_) => println!("{:#?}", client.log().await),
+        ClientCommands::Log(_) => {
+            for entry in client.log().await?.into_iter() {
+                let level = match entry.level {
+                    sdk::api::log::Level::Critical => "CRIT",
+                    sdk::api::log::Level::Error => "ERRO",
+                    sdk::api::log::Level::Warning => "WARN",
+                    sdk::api::log::Level::Info => "INFO",
+                    sdk::api::log::Level::Debug => "DEBG",
+                    sdk::api::log::Level::Trace => "TRCE",
+                };
+
+                println!(
+                    "{timestamp} {level} {msg}",
+                    timestamp = chrono::Local::now().format("%b %d %H:%M:%S%.3f"),
```

Comment:
We **shoud not** be using `Local::now` here, we should be using the time on the `entry`.

Reply:
Sure, i've noticed that too.